vizro-core/src/vizro/tables/dash_aggrid.py

```diff
@@ -0,0 +1,78 @@
+"""Module containing the standard implementation of `dash-ag-grid.AgGrid`."""
+
+import dash_ag_grid as dag
+import pandas as pd
+
+from vizro.models.types import capture
+from vizro.tables._utils import _set_defaults_nested
+
+FORMAT_CURRENCY_EU = """d3.formatLocale({
```

Comment:
This doesn't seem to be used?

Reply:
Yes it is, it get's used in the `DATA_TYPE_DEFINITIONS` just below, but for clarity I separated the two